internal/api/client/run.go

```diff
@@ -0,0 +1,87 @@
+package client
+
+import (
+	"context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"goph-keeper/internal/api/client/cli"
+	auth2 "goph-keeper/internal/api/client/handlers/auth"
+	"goph-keeper/internal/api/client/handlers/save"
+	"goph-keeper/internal/services/client/auth_client"
+	"goph-keeper/internal/services/client/binary_data_client"
+	"goph-keeper/internal/services/client/cards_client"
+	"goph-keeper/internal/services/client/credentials_client"
+	"goph-keeper/internal/services/client/get_all_data"
+	"goph-keeper/internal/services/client/text_data_client"
+	"goph-keeper/internal/storage/sqlite"
+	"log/slog"
+	"os"
+)
+
+const host = "localhost:8081"
```

Comment:
Я бы предложил сделать название более информативнывм, в противном случае оно теряется. 
```suggestion
const defaultHost = "localhost:8081"
```

Reply:
исправил